src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,42 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const currentTab = tabs.find(tab => selectedTabId === tab.id) || tabs[0];
+
+  const handleTabClick = (tab) => {
+    if (currentTab.id !== tab.id) {
+      onTabSelected(tab);
+    }
+  };
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              className={cn({
+                'is-active': currentTab.id === tab.id,
+              })}
+              data-cy="Tab"
+              key={tab.id}
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={() => handleTabClick(tab)}
```

Comment:
```suggestion
                onClick={handleTabClick(tab)}
```
we can avoid to create arrow function in jsx.